connector/src/main/java/org/geysermc/connector/network/translators/java/JavaLoginPluginRequestTranslator.java

```diff
@@ -25,17 +25,66 @@
 
 package org.geysermc.connector.network.translators.java;
 
+import com.github.steveice10.mc.auth.exception.request.RequestException;
+import com.github.steveice10.mc.auth.service.SessionService;
+import com.github.steveice10.mc.protocol.packet.login.client.LoginPluginResponsePacket;
+import com.github.steveice10.mc.protocol.packet.login.server.LoginPluginRequestPacket;
+import com.github.steveice10.packetlib.tcp.io.ByteBufNetInput;
+import io.netty.buffer.Unpooled;
+import lombok.SneakyThrows;
 import org.geysermc.connector.network.session.GeyserSession;
 import org.geysermc.connector.network.translators.PacketTranslator;
 import org.geysermc.connector.network.translators.Translator;
-
-import com.github.steveice10.mc.protocol.packet.login.client.LoginPluginResponsePacket;
-import com.github.steveice10.mc.protocol.packet.login.server.LoginPluginRequestPacket;
+import org.geysermc.connector.utils.LanguageUtils;
+import org.geysermc.cumulus.ModalForm;
+import org.geysermc.cumulus.response.ModalFormResponse;
 
 @Translator(packet = LoginPluginRequestPacket.class)
 public class JavaLoginPluginRequestTranslator extends PacketTranslator<LoginPluginRequestPacket> {
+    @SneakyThrows
```

Comment:
Can we not just handle errors properly instead of throwing them? Just a nice `session.getConnector().getLogger().error("Some message", e);`